execution_engine/src/core/execution/executor.rs

```diff
@@ -64,7 +61,6 @@ macro_rules! on_fail_charge {
             Ok(res) => res,
             Err(e) => {
                 let exec_err: Error = e.into();
-                warn!("Execution failed: {:?}", exec_err);
```

Comment:
@mpapierski @EdHastingsCasperLabs I've removed those warnings that were logged after user contract failure.

Reply:
Why?

Reply:
This is a user error returned when executing its contract. For example
```
Execution failed: Revert(ApiError::User(1) [65537])
```
We had removed logging of that in all other places, those were the few last ones that were not caught.

Reply:
This will be not only wrong – this is a user error that gets logged in the node and only node software should log, but also bad – for example on Delta, people send thousands of invalid deploys per second. Every one of those deploys will fail and will be logged. On every machine in the network.

Again, like I said, we had already been through this and we agreed to remove it. That's why there's no other place where we do log it, this seems to be some leftover that we missed.

That's why I asked @mpapierski and @EdHastingsCasperLabs for a review. They were part of the discussion and decision making.

Reply:
> 
> 
> Rather than remove them, could we consider changing them to `info!` or `debug!`?

I agree with @henrytill here I'd rather lower verbosity. If it's still to noisy why not silence execution-engine related logs through `RUST_LOG` if that's possible?

Reply:
Reverted all the changes to all EE files.